src/test/regress/expected/insert_select_single_shard_table_0.out

```diff
@@ -0,0 +1,845 @@
+--
```

Comment:
Can you please share the reason why we end-up with multiple files? I'd rather reduce the log level and remove the additional file if needed